MvvmCross/ViewModels/MvxAppStart.cs

```diff
@@ -3,32 +3,47 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Threading;
-using System.Threading.Tasks;
 using MvvmCross.Exceptions;
 using MvvmCross.Logging;
 using MvvmCross.Navigation;
 
 namespace MvvmCross.ViewModels
 {
-    public class MvxAppStart<TViewModel>
```

Comment:
Whilst we're adjusting this, is it worth creating a generic version of IMvxAppStart eg IMvxAppStart<TViewModel> : IMvxAppStart?

Reply:
Sorry but I don't get this one 🤔